src/CFamily/CMake/Resources.resx

```diff
@@ -117,17 +117,26 @@
   <resheader name="writer">
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
+  <data name="BadCMakeSettings" xml:space="preserve">
+    <value>[CompilationDatabaseLocator] Could not parse CMakeSettings.json: {0}</value>
+  </data>
+  <data name="BadCompilationDatabaseFile" xml:space="preserve">
+    <value>[CompilationConfigProvider] Could not parse compilation database: {0}</value>
+  </data>
   <data name="FoundCompilationDatabaseFile" xml:space="preserve">
-    <value>[CompilationDatabaseLocator] Reading compilation database from '{0}</value>
+    <value>[CompilationConfigProvider] Reading compilation database from '{0}</value>
   </data>
   <data name="NoBuildConfigInCMakeSettings" xml:space="preserve">
     <value>[CompilationDatabaseLocator] Specified build configuration '{0}' could not be found at '{1}'</value>
```

Comment:
Not changed in this PR, but this is a non-debug message, so the `[CompilationDatabaseLocator]` prefix won't be meaningful to the user (same comment for new line #139).

Reply:
Changed to [CMake] for all the info messages